config/database.yml

```diff
@@ -5,8 +5,8 @@ postgres: &postgres
   pool: 5
   host: <%= ENV["DEV_DATABASE_HOST"] || "localhost" %>
   port: 5432
-  username: exercism
-  password: apples
+  username: tetchi-mini
+  password:
```

Comment:
yeah sorry, it was getting late and didn't realize I committed a bunch of stuff that I didn't mean to lol (including compiled CSS). Fixing later today.